website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Scorecard (alpha)
+weight: 25
+---
+
+# operator-sdk scorecard (alpha)
+
+## Overview
+
+The scorecard command, part of the operator-sdk, executes tests on your operator based upon a user defined configuration file and test images.
```

Comment:
Can you split these lines up so they're ~100 characters or below? It's easier to review going forward.

Reply:
yep, just split the lines to be more human readable.  The tables I found however still had to have flowing lines, not sure if there is a way around that.